### DISCUSSION TITLE
Creators???

### DISCUSSION THREAD

**Discussion by CT-7567-REX** [Hash-Studios/Prism#236] [Q&A] [answered]
I wanna search for a specific creators but don't know how to find them. Is there anyone who know how to search for a specific creator

**Comment by codenameakshay** [comment]
Currently, there is now way to search for creators, but we are working on new version, so soon we will add this.

**Comment by admin** [comment, marked as answer]
this is live now, you can search for creators via username or names in v3.0.6 onwards
